.releaserc

```diff
@@ -2,8 +2,19 @@
   "prepare": [
     "@semantic-release/npm",
     {
+      "//": "build alpine binary inside an alpine container, or else the binary is broken",
       "path": "@semantic-release/exec",
-      "cmd": "npm i -g pkg && pkg . && shasum -a 256 snyk-to-html-linux > snyk-to-html-linux.sha256 && shasum -a 256 snyk-to-html-macos > snyk-to-html-macos.sha256 && shasum -a 256 snyk-to-html-win.exe > snyk-to-html-win.exe.sha256"
+      "cmd": "docker run --rm -ti -v `pwd`:/snyk node:alpine sh /snyk/scripts/alpine-build.sh"
```

Comment:
also, maybe `:/snyk` to `:/snyk-to-html` just to call the project folder by the real app name?